### PR TITLE
Corrected some spelling mistakes in Marathi language.

### PR DESCRIPTION
**Pull request by konprtk**
Adjusted some Marathi words and few mistakes.

### CODE REVIEW

**Review by prem-k-r — APPROVED** [review]

**Review by itz-rj-here — APPROVED** [review]
Idk marathi language. But the language changing is looking ok for me 🙂

File: languages.js [hunk 1]
```diff
@@ -1519,7 +1519,7 @@ const translations = {
         "gemini": "जेमिनी",
         "copilot": "कोपायलट",
         "perplexity": "पर्प्लेक्सिटी",
-        "firefly": "अॅडोबी फायरफ्लाय",
+        "firefly": "अडोबी फायरफ्लाय",
```
Comment (prem-k-r) [inline languages.js:1522]: Shouldn't this be अ‍ॅडोबी?

**Comment by konprtk** [comment]
Hi there,

Thank you for the review and approval.

As a native Marathi speaker from Maharashtra, I appreciate your attention
to the language details.

Best regards,
Pratik Lokhande

File: languages.js (hunk 1 above)
Comment (konprtk) [reply languages.js:1522]: chatGPTResponseOnHowToWriteAdobeFireflyInMarathiTranslation

check this

Comment (konprtk) [reply languages.js:1522]: Check this 

chatGPTResponseOnHowToWriteAdobeFireflyInMarathiTranslation

Comment (prem-k-r) [reply languages.js:1522]: IMG_20241204_115723

We'll proceed with whatever you decide 😀

Comment (konprtk) [reply languages.js:1522]: image


😁

Comment (itz-rj-here) [reply languages.js:1522]: Can you guys now stop?!! ChatGPT also includes this too.
image

Comment (konprtk) [reply languages.js:1522]: bro, is [XengShi](https://github.com/XengShi) Chinese?

Comment (konprtk) [reply languages.js:1522]: > Can you guys now stop?!! ChatGPT also includes this too. image

okay bro

Comment (prem-k-r) [reply languages.js:1522]: > image
> 
> 😁

Lol.. bro just your confirmation was enough

Comment (XengShi) [reply languages.js:1522]: > bro, is [XengShi](https://github.com/XengShi) Chinese?

North Korean

Comment (konprtk) [reply languages.js:1522]: > > bro, is [XengShi](https://github.com/XengShi) Chinese?
> 
> North Korean

Bro, I am also from North Korea, and I ran away in 2013
When did you run away then?

Comment (itz-rj-here) [reply languages.js:1522]: > > bro, is [XengShi](https://github.com/XengShi) Chinese?
> 
> North Korean

Seriously 🙂

**Comment by prem-k-r** [comment]
Hi @konprtk bro, please check out https://github.com/XengShi/materialYouNewTab/discussions/195#discussioncomment-11504630
Looking forward to your input if you'd like to contribute further and if it aligns with your interest..

**Pull request merged**